music_assistant/providers/spotify/provider.py

```diff
@@ -267,6 +336,40 @@ async def get_playlist(self, prov_playlist_id: str) -> Playlist:
         playlist_obj = await self._get_data(f"playlists/{prov_playlist_id}")
         return parse_playlist(playlist_obj, self)
 
+    # Podcast methods (delegate to podcast manager)
```

Comment:
not sure I see the real benefit of the podcasts helper here

Reply:
I did find two methods which weren't value adding at all and removed them.